src/main/scala/com/adobe/api/platform/runtime/metrics/KamonConsumer.scala

```diff
@@ -56,9 +56,28 @@ object KamonConsumer {
       .collect { case e if e.eventType == Activation.typeName => e } //Look for only Activations
       .foreach { e =>
         val a = e.body.asInstanceOf[Activation]
-        Kamon.histogram("waitTime", MeasurementUnit.time.milliseconds).refine("name" -> a.name).record(a.waitTime)
-        Kamon.counter("activations").refine("name" -> a.name).increment()
-        println(a.name)
+        val (namespace, action) = getNamespaceAction(a.name)
+
+        Kamon.counter("openwhisk.counter.container.activations").refine(("namespace" -> namespace),("action" -> action),("kind" -> a.kind),("memory" -> a.memory.toString),("status" -> a.statusCode.toString)).increment()
+        Kamon.histogram("openwhisk.histogram.container.memory").refine(("namespace" -> namespace),("action" -> action),("kind" -> a.kind)).record(a.memory)
```

Comment:
Note that `memory` here is the one specified in action deployment. So pretty much constant for specific action. So may be better to have it aggregated at global level